### DISCUSSION TITLE
Numba Timing.py Error

### DISCUSSION THREAD

**Discussion by quatronics613** [openai/whisper#1311] [Q&A] [answered]
I am slowly getting Whisper up and running.  I have worked through several installation challenges and now I am up this run-time challenge.  My code is the simple example:

```
import whisper

model = whisper.load_model("base")
result = model.transcribe("audio.mp3", fp16=False)
print(result["text"])
```

I have an audio.mp3 file.  The fp16=False was added to remediate an error.  Now I get this error:

```
/home/ubuntu/.local/lib/python3.10/site-packages/whisper/timing.py:58: NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.
  def backtrace(trace: np.ndarray):
Killed
```

Any help would be appreciated!

J

**Comment by jongwook** [comment, marked as answer]
The numba warning is just a warning (handled in #1233) and probably is not the reason why the process was killed. It's difficult to guess the reason only from `Killed` in the log, but the process going out of memory is a common reason. Running:

```
dmesg | grep -i kill
```

may give a better clue on why it was killed by the OS.

**Reply by quatronics613** [reply]
Thank you, joonwook!  That was it.
